### PR TITLE
Fix delta import commit notification

### PR DESCRIPTION
**Pull request by zoliMolnar**
This pull request fixes the "no commit notification" after delta import is finished bug.

### CODE REVIEW

File: snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/japi/branches/BranchTimestampUpdateTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		final String importId = lastPathSegment(createImport(importConfiguration).statusCode(201)
+				.extract().header("Location"));
+
+		getImport(importId).statusCode(200).body("status", equalTo(ImportStatus.WAITING_FOR_FILE.name()));
+		uploadImportFile(importId, getClass(), fileName).statusCode(204);
+		waitForImportJob(importId).statusCode(200).body("status", equalTo(ImportStatus.COMPLETED.name()));
+	}
+	
+	@Test
+	public void testHeadTimestampUpdateAfterImporting() {
```
Comment (nagyo) [inline snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/japi/branches/BranchTimestampUpdateTest.java]: Please move this to com.b2international.snowowl.snomed.api.rest.io.SnomedImportApiTest. There is no need to have this separately from all the other import tests.

Comment (nagyo) [reply snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/japi/branches/BranchTimestampUpdateTest.java]: You'll have everything you need available in that package, both the import archive and all the supporting methods. In the end you can remove this class and the duplicate archive as well.

File: snomed/com.b2international.snowowl.snomed.importer.rf2/src/com/b2international/snowowl/snomed/importer/rf2/util/ImportUtil.java [hunk 2]
```diff
@@ -437,31 +441,40 @@ public void run() {
...
 		try { 
-
-			final long lastCommitTime = CDOServerUtils.getLastCommitTime(branch);
-			context.setCommitTime(lastCommitTime);
 			
-			final SnomedCompositeImporter importer = new SnomedCompositeImporter(IMPORT_LOGGER, repositoryState, context, importers, ComponentImportUnit.ORDERING);
+			long commitTime = CDOServerUtils.getLastCommitTime(branch);
+			importContext.setCommitTime(commitTime);
+			
+			final SnomedCompositeImporter importer = new SnomedCompositeImporter(IMPORT_LOGGER, repositoryState, importContext, importers, ComponentImportUnit.ORDERING);
```
Comment (nagyo) [inline snomed/com.b2international.snowowl.snomed.importer.rf2/src/com/b2international/snowowl/snomed/importer/rf2/util/ImportUtil.java]: Why have you changed these three lines?

**Review by nagyo — CHANGES_REQUESTED** [review]

File: snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/rest/io/SnomedImportApiTest.java [hunk 3]
```diff
@@ -203,5 +226,41 @@ public void import11ExtensionConceptWithVersion() {
...
+				.build(repositoryId)
+				.execute(getBus())
+				.getSync(1, TimeUnit.MINUTES);
+		
+		IBranchPath path = BranchPathUtils.createPath(branchAfterImport.path());
+		
+		getComponent(path, SnomedComponentType.CONCEPT, "301795004").statusCode(200);
+		
+		
+		assertThat(branchBeforeImport.baseTimestamp(), not(equalTo(branchAfterImport.headTimestamp())));
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/rest/io/SnomedImportApiTest.java]: Use a shorter version of assertion to increase readability: `assertNotEquals(branchAfterImport.baseTimestamp(), branchAfterImport.headTimestamp())`.

**Review by cmark — APPROVED** [review]

**Pull request closed without merging**